actors/runtime/exitcode/reserved.go

```diff
@@ -92,9 +92,7 @@ const (
 	SysErrorReserved3 = ExitCode(12)
 	SysErrorReserved4 = ExitCode(13)
 	SysErrorReserved5 = ExitCode(14)
-
-	// Indicates something broken within the VM.
-	SysErrInternal = ExitCode(15)
```

Comment:
I'd keep the definition, maybe rename to something indicating it's a VM error

Reply:
The problem is that this can never safely be used. These exit codes go on chain, but every time there's a VM error of this form (idiosyncratic, not reproducible between impls) it should *not* go on chain; instead the state evaluation should just fail.

It should never have existed. If Lotus still uses it, they are all bugs that should either be a different exit code, or a "fatal" error.